include/deal.II/fe/fe_interface_values.h

```diff
@@ -450,6 +462,10 @@ class FEInterfaceValues
    * as determined in reinit().
    */
   FEFaceValuesBase<dim> *fe_face_values_neighbor;
+
+#ifdef DEBUG
+  bool is_level_cell;
+#endif
```

Comment:
Let's not do this. We've tried to keep the debug and release libraries ABI compatible -- they may contain different code, but the sizes of structures and interfaces of functions should be the same. So just add the variable if you need it.